test/e2e-v2/cases/storage/banyandb/docker-compose.yml

```diff
@@ -0,0 +1,121 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  banyandb:
+    image: "skywalking-banyandb:${TAG}"
+    command: ["standalone", "--root-path", "/tmp/banyandb-data"]
+    expose:
+      - 17912
+    ports:
+      - 17912
+      - 6060
+    networks:
+      - e2e
+    healthcheck:
+      test: ["CMD", "nc", "-zv", "127.0.0.1", "17912"]
+      interval: 5s
+      timeout: 60s
+      retries: 120
+
+  oap:
+    # Pin OAP server
+    image: "lujiajing1126/skywalking-oap:${SW_OAP_COMMIT}"
+    expose:
+      - 11800
+      - 12800
+      - 10051
+      - 5005
+    ports:
+      - 12800
+    networks:
+      - e2e
+    volumes:
+      - ./../prepare/setup-oap/log4j2.xml:/skywalking/config/log4j2.xml
+    environment:
+      SW_CLUSTER_ZK_HOST_PORT: zk:2181
+      SW_CONFIG_ETCD_PERIOD: 1
+      SW_CONFIG_ETCD_ENDPOINTS: http://etcd:2379
+      SW_CLUSTER_ETCD_ENDPOINTS: http://etcd:2379
+      SW_STORAGE: banyandb
+      SW_STORAGE_BANYANDB_HOST: banyandb
+    healthcheck:
+      test: [ "CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/11800" ]
+      interval: 5s
+      timeout: 60s
+      retries: 120
+    depends_on:
+      banyandb:
+        condition: service_healthy
+
+  provider:
+    image: "quay.io/megrez/skywalking-oap-e2e-provider:latest"
```

Comment:
As we need to pend on @hanahmily 's measure side works, could you work on main repo to support e2e test service images available?

Reply:
Sure. I could submit another PR to publish these images in the main repo.

Reply:
https://github.com/apache/skywalking/pkgs/container/skywalking%2Fe2e-service-* should be a good prefix for your image.

Reply:
> https://github.com/apache/skywalking/pkgs/container/skywalking%2Fe2e-service-* should be a good prefix for your image.

Got it